### PR TITLE
chore(docs): Add examples to `Float32` module

### PR DESCRIPTION
**Pull request by spotandjake**
This improves the docs in the `Float32` module, It adds examples for the float32 syntax to the top following the pattern ive been using in other libs, Adds example to all the functions. and updates the documentation for `Float32.infinity` and `Float32.nan` to mention the alternative syntax.

### CODE REVIEW

**Review by alex-snezhko — APPROVED** [review]

File: stdlib/float32.gr [hunk 1]
```diff
@@ -25,6 +30,7 @@ let _VALUE_OFFSET = 4n
 
 /**
  * Infinity represented as a Float32 value.
+ * This is equivalent to `Infinityf`.
```
Comment (phated) [inline stdlib/float32.gr:33]:
```suggestion
 * This is an alternative to the `Infinityf` literal.
```

File: stdlib/float32.gr [hunk 2]
```diff
@@ -33,6 +39,7 @@ provide let infinity = Infinityf
 
 /**
  * NaN (Not a Number) represented as a Float32 value.
+ * This is equivalent to `NaNf`.
```
Comment (phated) [inline stdlib/float32.gr:42]:
```suggestion
 * This is an alternative to the `NaNf` literal.
```

NaN is not equivalent to NaN

**Review by phated — CHANGES_REQUESTED** [review]
Same as the other PR

**Comment by spotandjake** [comment]
Made those changes

**Review by phated — APPROVED** [review]

**Pull request merged**